src/test/java/pl/touk/sputnik/connector/saas/SaasFacadeTest.java

```diff
@@ -0,0 +1,72 @@
+package pl.touk.sputnik.connector.saas;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import pl.touk.sputnik.HttpConnectorEnv;
+import pl.touk.sputnik.configuration.Configuration;
+import pl.touk.sputnik.configuration.ConfigurationSetup;
+import pl.touk.sputnik.connector.FacadeConfigUtil;
+import pl.touk.sputnik.review.*;
```

Comment:
[Checkstyle] ERROR: Using the '.*' form of import should be avoided - pl.touk.sputnik.review.*.